### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/961
https://github.com/open-orchestra/open-orchestra-model-interface/pull/98
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/386
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/111
https://github.com/open-orchestra/open-orchestra-front-demo/pull/67
https://github.com/open-orchestra/open-orchestra-provision/pull/15
https://github.com/open-orchestra/open-orchestra-docs/pull/66

### CODE REVIEW

File: ApiBundle/Controller/NodeController.php [hunk 1]
```diff
@@ -80,6 +89,23 @@ public function showOrCreateAction(Request $request, $nodeId)
...
+     * @Config\Method({"GET"})
+     * @Api\Serialize()
+     *
+     * @Config\Security("has_role('ROLE_ACCESS_TREE_NODE')")
+     *
+     * @return FacadeInterface
+     */
+    public function showOrCreateErrorAction(Request $request, $nodeId)
+    {
+        return $this->showOrCreateAction($request, $nodeId, true);
```
Comment (nicolasThal) [inline ApiBundle/Controller/NodeController.php]: This method is quite useless, you could directly put the route on the other method and adding a default value for the `$errorNode`

File: Backoffice/LeftPanel/Strategies/ErrorPagesPanelStrategy.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+namespace OpenOrchestra\Backoffice\LeftPanel\Strategies;
+
+use OpenOrchestra\ModelInterface\Model\ReadNodeInterface;
+
+/**
+ * Class ErrorPagesPanelStrategy
+ */
+class ErrorPagesPanelStrategy extends AbstractLeftPanelStrategy
+{
+    const ROLE_ACCESS_TREE_NODE = 'ROLE_ACCESS_TREE_NODE';
```
Comment (nicolasThal) [inline Backoffice/LeftPanel/Strategies/ErrorPagesPanelStrategy.php]: This const is already existing somewhere I think

Comment (itkg-ngilain) [inline Backoffice/LeftPanel/Strategies/ErrorPagesPanelStrategy.php]: It is also defined in TreeNodePanelStrategy, but I'm not fan of one strategy refering to another. Alternativly we could define that const in the AbastractStrategy if you're ok.

Comment (nicolasThal) [inline Backoffice/LeftPanel/Strategies/ErrorPagesPanelStrategy.php]: As the string it refers to is the same, i think that you can refer to another strategy.

Or you could change the name ROLE_ACCESS_TREE_NODE to ROLE_ACCESS_ERROR_TREE_NODE

**Comment by itkg-ngilain** [comment]
@nicolasThal : error pages moved from ErrorPanelStrategy to TreeNodePaneStrategy

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**